modules/nanopolish/eventalign/main.nf

```diff
@@ -0,0 +1,35 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName } from './functions'
+
+params.options = [:]
+def options    = initOptions(params.options)
+
+process NANOPOLISH_EVENTALIGN {
+    publishDir "${params.outdir}",
+        mode: params.publish_dir_mode,
+        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), publish_id:'meta.id') }
+
+    conda     (params.enable_conda ? "bioconda::nanopolish==0.13.2" : null)
+    container "quay.io/biocontainers/nanopolish:0.13.2--he3b7ca5_2"
+
+    input:
+    tuple val(meta), path(bam), path(bai)
+    path fast5
+    path fastq
+    path fasta
+    path gtf
```

Comment:
Ah, I misunderstood the example command from the link above and assumed it was required but not according to this: https://nanopolish.readthedocs.io/en/latest/manual.html#event-align. Just FYI, `--scale-events`: `scale events to the model, rather than vice-versa`.